src/FourierFlows.jl

```diff
@@ -89,15 +89,18 @@ include("timesteppers.jl")
 include("diffusion.jl")
 
 # Import CUDA utilities if cuda is detected.
-const HAVE_CUDA = try
-    using CuArrays
-    true
-catch
-    false
+if has_cuda()
+  try
+    using CuArrays # we have CUDA, so this should not fail
+  catch ex
+    # something is wrong with the user's set-up (or there's a bug in CuArrays)
+    @warn "CUDA is installed, but CuArrays.jl fails to load" exception=(ex, catch_backtrace())
```

Comment:
Hmm, do we want to throw this warning? I feel the majority of users will not be using cuda functionality; we don't need to be spamming them all with warnings about `CuArrays`... ?

Reply:
Indeed; drop the warning.

Reply:
Btw, on my laptop (without GPU) I don't get any warning.

Reply:
Oh, I was confused. This warning is only thrown if `has_cuda() == true`, but `CuArrays` errors. Since `has_cuda() == false` on your laptop there's no issue. Ok, so this PR is good to merge.